pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/ReturnFromFinallyBlock.xml

```diff
@@ -57,4 +57,63 @@ public class Foo {
 }
      ]]></code>
     </test-code>
+    <test-code>
+        <description><![CDATA[
+Return from a Class in Finally
+    ]]></description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+public class Foo {
+    String bar() {
+    try {
+    } finally {
+        Object o = new Object() {
+            @Override
+            public String toString() {
+                return "";
+            }
+        };
+    }
+  }
+}
+        ]]></code>
+    </test-code>
+    <test-code>
+        <description><![CDATA[
+Return from Lambda in Finally
```

Comment:
We usually refer to the issue number, that this test case tests, so that we might have a look later on, why we added this test case. So, here, you can just use this description:

```
#1035 [java] ReturnFromFinallyBlock: False positive on lambda expression in finally block
```

This also works without CDATA.